source/changelog.txt

```diff
@@ -403,8 +403,7 @@ New features in this release:
 - Autocomplete is now aware of the ``--apiStrict`` flag. When
   ``--apiStrict`` is ``true``, autocomplete only completes methods
   that work with your defined API version. For more information, see
-  `Versioned API
-  <https://docs.mongodb.com/v5.0/reference/versioned-api/>`__.
+  :manual:`Versioned API </reference/stable-api/>`.
```

Comment:
I think the link text should be "Stable API". Justification in my mind is that users will click this link and arrive on a page called "Stable API". If the link text doesn't match the target page text, users might get confused or think they've arrived in the wrong place.

Reply:
Thanks Jeff! Solid justification; updated! :-)